dora/src/vm/functions.rs

```diff
@@ -118,6 +119,75 @@ impl Fct {
         }
     }
 
+    #[allow(dead_code)]
+    pub fn new_replacement(
```

Comment:
I would try to avoid adding a second method. I suspect this method was copy&pasted from above? I would suggest simply combining both methods. Should be possible if we would use e.g. `has_override: ast.has_override || annotations.contains(...)`? (Let's use a local variable for that `annotations.contains(...)` though. Then it should work for the old & new representation and after we've migrated e.g. `@override` fully, we can simply remove that `ast.has_override ||` part.

Reply:
good idea! will do.

Reply:
> I suspect this method was copy&pasted from above?

I think when I made these additions, the current method didn't exist.

Reply:
I reduced the scope of this commit, checking for annotations in `AnnotationUsages` requires having the annotations registered first.